apps/api/src/app/auth/services/auth.service.ts

```diff
@@ -11,14 +11,14 @@ import {
   UserRepository,
 } from '@novu/dal';
 import { AuthProviderEnum, IJwtPayload, ISubscriberJwt, MemberRoleEnum, SignUpOriginEnum } from '@novu/shared';
+import { AnalyticsService } from '@novu/application-generic';
```

Comment:
I've moved the `AnalyticsService` from the `API` app to the `application-generic` package. The service is rn used in WS and API apps.